### DISCUSSION TITLE
Noob question. GitHub Actions and Wix.

### DISCUSSION THREAD

**Discussion by sblantipodi** [wixtoolset/issues#8873] [Questions]
Hi there.
I'm using Wix 3 to generate a .exe file using GitHub Actions.

It was easy to do it with Wix3 using the Wix Installer, 
I copied all the required files for wix in a folder and then I used those files.
How can I do it with Wix 5?
I don't see an installer for Wix 5 and I don't know how to obtain the wix.exe file. 

I would like to bundle the wix.exe on my repo to make it usable by my GitHub action
without using dotnet. Is this possible?

**Comment by barnson** [comment]
Just use [MSBuild](https://wixtoolset.org/docs/intro/#msbuild) and you don't have to install anything.

**Reply by sblantipodi** [reply]
I need it for jpackage using GitHub Actions. 
How can I use msbuild for it?

**Reply by barnson** [reply]
You should check with the JPackage folks whether they support WiX v5 yet.
